code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -90,7 +93,7 @@
 	fire_delay = 7
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/riot
 	sawn_desc = "Come with me if you want to live."
-	unique_reskin = list("Tatical" = "riotshotgun",
```

Comment:
Why are you braking the sprites?

Reply:
that...shouldn't? tactical was misspelled this whole time and this should just fix the grammar on it and not break the associated sprite.

Reply:
99% sure it relies on the second value